modules/updating-sno.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-cluster-between-minor.adoc
+// * updating/updating-cluster-cli.adoc
+// * updating/updating-cluster.adoc
+
+= Updating single node {product-title}
+
+You can update, or upgrade, a single-node {product-title} cluster using either the console or CLI. 
+
+However, note the following: 
+
+* The prerequisite to pause the `MachineHealthCheck` resources is not required, because there is no other node to perform the health check.
+
+* Restoring a single-node {product-title} cluster using an etcd backup is not officially supported. However, it is good practice to perform the etcd backup in case your upgrade fails. You might be able to restore your cluster to a previous state using the backup if your control plane is healthy.  
```

Comment:
```suggestion
* Restoring a single-node {product-title} cluster by using an etcd backup is not officially supported. However, it is good practice to perform the etcd backup in case your upgrade fails. If your control plane is healthy, you might be able to restore your cluster to a previous state by using the backup.  
```